.github/workflows/ci.yml

```diff
@@ -147,7 +147,7 @@ jobs:
           conda list
       - run: sudo apt update && sudo apt install -y libhdf5-dev
       - name: Api tests
-        run: python -m pytest modin/pandas/test/test_api.py
+        run: python -m pytest modin/test/pandas/test_api.py
```

Comment:
Do we want to put all test directories in one place (modin/config/test, modin/core/execution/dispatching/factories/test, modin/experimental/core/execution/native/implementations/omnisci_on_native/test, etc.)?

Reply:
@YarShev I think that will be best, yes.

Reply:
Hi, so i guess I should continue refactoring with all folders (i wanted to check if its fine with that one folder, but I guess thats not the best workflow, especially for this issue) and after I will update the PR and then it makes sense to check for the modification with the CI and test pipeline?

Reply:
@teaVeloper Yes, I think that would be good. It looks great so far!